TODO.md

```diff
@@ -0,0 +1,21 @@
+# TODO
+- TODO MVP
+
+## TODO MVP
+- TODO APP
+- TODO UI
+- TODO SERVICES
+
+## TODO APP
+- FEATURE: update bin/[app] to take options per CB0, ie. set center to options.lat, options.lon
+- FEATURE: add args for liveDataURL [string: url] and liveDataType [string: wss url | sse url | ...],
+  - eg. send old geoy data to mapscii via SEE
+- RESEARCH: options for tileSource
+- RESEARCH: stories: animated map + espeak TTS script
+- RESEARCH: add narrative/realtime detail to footer
+
+## TODO UI
+- TBD
+
+## TODO SERVICES
+- FEATURE: use app-stack
```

Comment:
What are these entries? A personal work list?
Why add another todo list? `Readme.md` already has one.
Why keep todos in the project files? The list in `Readme.md` has not been updated in over 5 years.
Why not on GitHub to link Pull Requests, for discussions or for other people to pick them up?

In the repo they won't have easy-to-see metadata like an author or a timestamp.

Reply:
Agreed. Adding my TODO and README was not appropriate for a PR. Thanks